test/automation/src/sql/notebook.ts

```diff
@@ -372,31 +372,38 @@ export class NotebookView {
 	 * @returns tree item ids from Notebooks View
 	 */
 	async getNotebookTreeItemIds(): Promise<string[]> {
-		return (await this.code.waitForElements(NotebookView.notebookTreeItem, false)).map(item => item.attributes['id']);
+		return (await this.code.waitForElements(NotebookTreeView.notebookTreeItem, false)).map(item => item.attributes['id']);
+	}
+
+	/**
+	 * Helper function
```

Comment:
Would suggest actually describing what this function does. Something like `Gets the list of IDs of items in the 'Pinned Notebooks' tree`. That's a lot more useful than just "helper function"